server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -42,6 +43,16 @@ const accountMutations = {
           throw new Forbidden();
         }
 
+        if (
+          args.key === 'collectivePage' &&
+          ![AccountTypeToModelMapping.FUND, AccountTypeToModelMapping.PROJECT].includes(account.type)
+        ) {
+          const budgetSection = args.value.sections?.find(s => s.section === 'budget');
+          if (!budgetSection.isEnabled) {
```

Comment:
Isn't this going to crash if you don't provide a `section` budget (because `budgetSection` will be null)?

Reply:
Yes, already fixed here: https://github.com/opencollective/opencollective-api/pull/4221/commits/ed5361b9021ff7528b83cbb6f6c6bad836c9b78a